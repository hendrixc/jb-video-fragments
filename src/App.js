import React, { Component } from 'react';
import { ThemeProvider } from '@rmwc/theme';
import Home from './components/home/Home';
import colors from './style/colors';
import 'material-icons/iconfont/material-icons.scss';
import GlobalProvider from './context/GlobalProvider';

class App extends Component {
  render() {
    return (
      <ThemeProvider options={{
        primary: colors.primary800,
        secondary: colors.secondary500,
        onPrimary: colors.primary400,
        textPrimaryOnBackground: colors.black,
      }}
      >
        <GlobalProvider>
          <Home />
        </GlobalProvider>
      </ThemeProvider>
    );
  }
}

export default App;
