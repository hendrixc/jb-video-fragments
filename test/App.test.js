import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from '../src/App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders an home component into the App as snapshot', () => {
  const TextInputComponent = renderer.create(<App />).toJSON();
  expect(TextInputComponent).toMatchSnapshot();
});
