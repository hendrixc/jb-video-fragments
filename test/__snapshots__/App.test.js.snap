// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an Home component into the App as snapshot 1`] = `
<div
  className="App"
>
  <div
    className="Home"
  >
    <div
      className="Header"
    >
      <header
        className="mdc-toolbar"
      >
        <div
          className="mdc-toolbar__row"
        >
          <div
            className="mdc-toolbar__title"
          >
            Toolbar
          </div>
        </div>
      </header>
    </div>
  </div>
</div>
`;
